dali/base/dafdesc.cpp

```diff
@@ -3635,3 +3719,22 @@ IStoragePlane * getRemoteStoragePlane(const char * name, bool required)
     group.append(name).toLowerCase();
     return getStoragePlane(group, { "remote" }, required);
 }
+
+
+AccessMode getAccessModeFromString(const char *access)
+{
+    // use a HT?
+    if (streq(access, "read"))
+        return AccessMode::read;
+    else if (streq(access, "write"))
+        return AccessMode::write;
+    else if (streq(access, "random"))
+        return AccessMode::random;
+    else if (streq(access, "sequential"))
+        return AccessMode::sequential;
+    else if (streq(access, "noMount"))
+        return AccessMode::noMount;
+    else if (isEmptyString(access))
+        return AccessMode::none;
+    throwUnexpected();
```

Comment:
Should probably throw a more informative error.

Reply:
will change.